### PR TITLE
23 Installing EF Core to a computer

### PR DESCRIPTION
**Pull request by arynhuck**
This would close issue #23

### CODE REVIEW

File: content/efcore/getting-started/installing-ef-core.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Installing EF Core  
+ 
+In this lesson, you will learn how to install EF Core on your machine. You do not need to complete these steps to continue with the rest of the tutorial. 
+ 
+## Install .NET Core 
+ 
+If you haven't already, you will first need to install .NET Core on your machine. Go to the [.NET Core Page](https://www.microsoft.com/net/core) of the Microsoft website, and follow the installation instructions for your platform of choice. You can ensure .NET Core is installed by running the following command: 
+ 
+``` 
+dotnet –version
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: should be `--version`

File: content/efcore/getting-started/installing-ef-core.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+} 
+``` 
+ 
+To add EF Core functionality to the [.NET Core Command-Line Interface](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/), you need to add a package to the `tools` section of `project.json`: 
+ 
+``` 
+"tools": { 
+    "Microsoft.EntityFrameworkCore.Tools": "1.0.0-preview2-final" 
+} 
+
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: Remove extra newline

File: content/efcore/getting-started/toc.md [hunk 3]
```diff
@@ -1 +1,2 @@
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```
Comment (helmoski) [inline content/efcore/getting-started/toc.md]: I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`

File: content/efcore/getting-started/README.md [hunk 4]
```diff
@@ -12,4 +12,5 @@ This tutorial is aimed at people who are learning EF Core for the first time. Mo
 ## Topics Covered
 
 This tutorial will cover the following topics:
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```
Comment (helmoski) [inline content/efcore/getting-started/README.md]: I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`

File: content/efcore/getting-started/installing-ef-core.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Installing EF Core  
+ 
+In this lesson, you will learn how to install EF Core on your machine. You do not need to complete these steps to continue with the rest of the tutorial.
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: Just thought of this: We should probably state the reason they don't need to finish the steps in this lesson (The code examples in the tutorial can be run in-browser)

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/README.md (hunk 4 above)
Comment (arynhuck) [reply content/efcore/getting-started/README.md]: Yep. That is a thing we did not catch. 😐  _panics_

File: content/efcore/getting-started/toc.md (hunk 3 above)
Comment (arynhuck) [reply content/efcore/getting-started/toc.md]: done

File: content/efcore/getting-started/installing-ef-core.md (hunk 2 above)
Comment (arynhuck) [reply content/efcore/getting-started/installing-ef-core.md]: done

File: content/efcore/getting-started/installing-ef-core.md (hunk 1 above)
Comment (arynhuck) [reply content/efcore/getting-started/installing-ef-core.md]: done

File: content/efcore/getting-started/installing-ef-core.md (hunk 5 above)
Comment (arynhuck) [reply content/efcore/getting-started/installing-ef-core.md]: done

**Comment by arynhuck** [comment]
Adjustments made! :shipit: ?

File: content/efcore/getting-started/installing-ef-core.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+ 
+```  
+mkdir MyEfCoreProject # Create directory for project  
+cd MyEfCoreProject    # Navigate to project directory   
+dotnet new            # Initialize .NET Core project (generates Startup.cs and project.json)  
+```  
+  
+## Add EF Core to Project 
+ 
+
```
Comment (helmoski) [inline content/efcore/getting-started/installing-ef-core.md]: Remove extra newline

**Review by helmoski — CHANGES_REQUESTED** [review]

**Review by helmoski — APPROVED** [review]

**Comment by helmoski** [comment]
:shipit:

**Pull request merged**